CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [PR 144](https://github.com/salesforce/django-declarative-apis/pull/141) Allow Dependabot to update dev dependencies
 - [PR 143](https://github.com/salesforce/django-declarative-apis/pull/143) Testing: Replace `flake8` and `black` with `ruff`, add testing for Python 3.12, drop testing for Python 3.7
 - [PR 146](https://github.com/salesforce/django-declarative-apis/pull/146) Add ExpandableGeneric filter type
+- [PR 148](https://github.com/salesforce/django-declarative-apis/pull/148) chore: upgrade django 4.2 LTS (@W-14178534)
```

Comment:
Thank you for adding this.

It looks like we forgot to update this CHANGELOG the last time we released DDA.  This release contains the "ExpandableGeneric" filter type (and I assume the PRs before it):

[Release Django Declarative APIs v0.31.6 · salesforce/django-declarative-apis](https://github.com/salesforce/django-declarative-apis/releases/tag/v0.31.6)

This isn't your mistake, but would you mind fixing the CHANGELOG?

Reply:
You could potentially fix it in the PR when you make a release, so I'll go ahead and approve this PR.